src/sentry/conf/server.py

```diff
@@ -1058,10 +1056,6 @@ def create_partitioned_queues(name):
     "organizations:release-archives": False,
     # Enable the new release details experience
     "organizations:release-comparison": False,
-    # Enable the project level transaction thresholds
-    "organizations:project-transaction-threshold": False,
-    # Enable the transaction level thresholds
-    "organizations:project-transaction-threshold-override": False,
```

Comment:
Might be a good idea to delete the flags themselves in a follow up after all references to them have been removed so we don't experience any hiccups in some intermediate state.

Reply:
good idea